etc/dbus-serialbattery/battery.py

```diff
@@ -68,6 +68,8 @@ def __init__(self, port, baud, address):
         self.max_battery_charge_current = None
         self.max_battery_discharge_current = None
         self.has_settings = 0
+        self.max_battery_voltage = None
+        self.min_battery_voltage = None
```

Comment:
Now you are not calculating the min/max battery voltages anymore?

Reply:
Oops, sorry. This is bug, where is the place where these values can be calculated? At this point is still self.cell_count = None (I have to get used to python) Would you please give a tip on where this can be done? Don't the implementations of the BMS do that?

Reply:
No problem. Everyone has to start somewhen :-) I put it in the `init_values()` function, so you so more changes needed.